src/annotator/anchoring/range.js

```diff
@@ -382,7 +396,10 @@ export class SerializedRange {
         targetOffset--;
       }
 
-      for (let tn of getTextNodes($(node))) {
+      for (let tn of getTextNodes(node)) {
+        if (tn.nodeValue === null) {
+          continue;
```

Comment:
When is a text node's nodeValue null? Is it possible to add a test for this case?

Reply:
I looked this up in the DOM specs this morning and found that `nodeValue` is always a string for a `Text` node: https://dom.spec.whatwg.org/#dom-node-nodevalue.

In the case of `Text` nodes `nodeValue` is an alias for `data` which is always non-null. I've changed this to use `data` instead.